spec/filesystem.rb

```diff
@@ -75,6 +75,23 @@ def create_initial_directory_hierarchy
     FileUtils.touch Metalware::Constants::DEFAULT_CONFIG_PATH
   end
 
+  # Print every directory and file loaded in the FakeFS.
+  def debug!
+    begin
+      # This can fail oddly if nothing matches (see
+      # https://github.com/fakefs/fakefs/issues/371), hence the `rescue` with a
+      # simpler glob.
+      matches = Dir['**/*']
```

Comment:
If we want to print the entire directory structure, we can also use `Dir['/**/*']`. Not sure if that fixes the bug as we should always be able to find the root dir: `/`.